padertorch/configurable.py

```diff
@@ -864,22 +913,33 @@ def recursive_class_to_str(config, sort=False):
     >>> cfg = {'out_features': 2, 'in_features': 1, 'factory': torch.nn.Linear}
     >>> recursive_class_to_str(cfg, sort=True)
     {'factory': 'torch.nn.modules.linear.Linear', 'in_features': 1, 'out_features': 2}
+    >>> cfg = {'partial': torch.nn.LeakyReLU, 'negative_slope': 0.01, 'inplace': False}
+    >>> recursive_class_to_str(cfg, sort=True)
+    {'partial': 'torch.nn.modules.activation.LeakyReLU', 'negative_slope': 0.01, 'inplace': False}
+    >>> cfg = {'partial': torch.nn.LeakyReLU, 'inplace': False, 'negative_slope': 0.01}
+    >>> recursive_class_to_str(cfg, sort=True)
+    {'partial': 'torch.nn.modules.activation.LeakyReLU', 'negative_slope': 0.01, 'inplace': False}
+    >>> cfg = {'inplace': False, 'negative_slope': 0.01, 'partial': torch.nn.LeakyReLU}
+    >>> recursive_class_to_str(cfg, sort=True)
+    {'partial': 'torch.nn.modules.activation.LeakyReLU', 'negative_slope': 0.01, 'inplace': False}
 
     """
     # ToDo: Support tuple and list?
     if isinstance(config, dict):
         d = config.__class__()
-        if sort and 'factory' in config:
+        special_key = _get_special_key(config)
+
+        if sort and special_key:
             # Force factory to be the first key
-            d['factory'] = None  # will be set later
-            factory = import_class(config['factory'])
-            arg_names = inspect.signature(factory).parameters.keys()
+            d[special_key] = None  # will be set later
```

Comment:
The comment above this line still uses "factory", should be "special key"

Reply:
The following code is a little hard to understand. Maybe little longer comment that this is only done to ensure that the ordering of the entries equals the ordering of the signature could help